### DISCUSSION TITLE
Device Info cards - better support for things created in YAML

### DISCUSSION THREAD

**Discussion by SeanPM5** [home-assistant/frontend#11219] [Other feature requests]
## The request

On the new Device Info page cards, it is currently rather limited for users who have their automations, scripts, and scenes defined in YAML rather than the UI editors. Everything is shown grayed out and when you hover it there's a tooltip that you can't edit. That is fine and to be expected, but there's nothing else you can do either...

![image](https://i.imgur.com/tb137A7.png)

The actual panels don't have this problem as much - you can trigger scripts by pressing the play button to their left, you can activate scenes by pressing the play button to their left, and you can execute automations by pressing the more info button on the right. So those panels are still useful for YAML users.

Would be nice if something similar could be added here. I realize there's more limited width with these cards, but maybe change the behavior slightly so that uneditable things would bring up the more info dialog? That way you can just press activate/trigger/execute etc and it would still have some value for YAML people.

**Comment by zsarnett** [comment]
We should add a button to the more info page on each row
